src/main/java/io/vertx/core/http/impl/WebSocketFrameFactoryImpl.java

```diff
@@ -11,11 +11,16 @@
 
 package io.vertx.core.http.impl;
 
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import io.netty.buffer.UnpooledDirectByteBuf;
 import io.vertx.core.buffer.Buffer;
 import io.vertx.core.http.WebSocketFrame;
 import io.vertx.core.http.impl.ws.WebSocketFrameImpl;
 import io.vertx.core.spi.WebSocketFrameFactory;
 
+import java.nio.charset.Charset;
```

Comment:
this file does not seem to be modified, we need to avoid modifications in it (imports and  blank line)